controllers/apps/transformer_backup_policy_tpl.go

```diff
@@ -225,6 +230,96 @@ func (r *BackupPolicyTPLTransformer) buildBackupPolicy(policyTPL appsv1alpha1.Ba
 	return backupPolicy
 }
 
+// mergeClusterBackup merges the cluster backup configuration into the backup policy.
+func (r *BackupPolicyTPLTransformer) mergeClusterBackup(transCtx *ClusterTransformContext, cluster *appsv1alpha1.Cluster,
+	backupPolicy *dataprotectionv1alpha1.BackupPolicy) {
+
+	backupEnabled := func() bool {
+		return cluster.Spec.Backup != nil && boolValue(cluster.Spec.Backup.Enabled)
+	}
+
+	if backupPolicy == nil {
+		// backup policy is nil, can not enable cluster backup, so log and return.
+		if backupEnabled() {
+			// TODO: should we return an error here?
+			transCtx.Logger.Info("backup policy is nil, can not enable cluster backup",
+				"namespace", cluster.Namespace, "cluster", cluster.Name)
+		}
+		return
+	}
+
+	backup := cluster.Spec.Backup
+	spec := &backupPolicy.Spec
+	setSchedulePolicy := func(schedulePolicy *dataprotectionv1alpha1.SchedulePolicy, enable bool) {
+		if schedulePolicy == nil {
+			if enable {
+				// failed to find the schedule policy for backup method, so log and return.
+				// TODO: should we return an error here?
+				transCtx.Logger.Info(fmt.Sprintf("failed to find the schedule policy for backup method %s", backup.Method),
+					"namespace", cluster.Namespace, "cluster", cluster.Name)
+			}
+			return
+		}
+		schedulePolicy.Enable = enable
+		if enable && backup.CronExpression != "" {
+			schedulePolicy.CronExpression = backup.CronExpression
+		}
+	}
+
+	// disable automated backup, set all backup schedule to false
+	if !backupEnabled() {
+		setSchedulePolicy(spec.Schedule.Snapshot, false)
+		setSchedulePolicy(spec.Schedule.Datafile, false)
+		setSchedulePolicy(spec.Schedule.Logfile, false)
```

Comment:
logfile controlled by pitrEnabled

Reply:
Yes, but `backup.Enabled` is a switch to turn on/off the automated backup, if `backup.Enabled` is false, the PITR is also disabled.